public/index.html

```diff
@@ -5,10 +5,14 @@
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="theme-color" content="#000000" />
         <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+        {{{NAV_SCRIPTS}}} {{{NAV_MENU_RESOURCES}}} {{{NAV_STYLES}}}
```

Comment:
Disse erstattes med data fra `hentDekoratør`-funksjonen ved hjelp av mustache

Reply:
Kan være irriterende under utvikling fordi da vises ikke dekoratøren. Man kan isåfall slette disse under utvikling, bare ikke commit.

Reply:
I de andre appen (feks kontakt-oss) har vi hatt likt som dette 🙂 Vi har ikke giddet å bruke tid på å override dev serveren man bruker lokalt for å få fjerna lenkene... Man er jo mulig lm man ønsker det

Reply:
Tenkte bare at de som vil kan slette linjene lokalt